credit_card.rb

```diff
@@ -1,6 +1,6 @@
 # Class Comment
 class CreditCard
-  attr_reader :balance
+  attr_reader :balance,:payment
```

Comment:
We don't need a `attr_reader` for :payment.  This line should be `attr_reader :balance`.